### DISCUSSION TITLE
TransitionProps shaken out of existance?

### DISCUSSION THREAD

**Discussion by amery** [cdmoro/bootstrap-vue-3#887] [General]
I'm quite confused about this . I'm using 0.6.2 on nuxt3 and for some reason I'm getting the following error on the console

```
SyntaxError: The requested module '/_nuxt/@fs/home/amery/projects/xxxx/node_modules/.vite/deps/vue.js?v=ec154c92' does not provide an export named 'TransitionProps' (at BTransition.vue:9:26)
```

any idea what's going on?

**Comment by VividLemon** [comment]
Try adding Vue as a dependency if it isn't one

**Reply by amery** [reply]
@VividLemon I just added vue explicitly to the nuxt3 project (pnpm add vue; got 3.2.45). unfortunately `nuxt dev` still causes the same problem :(

**Reply by VividLemon** [reply]
It's possible that TransitionProps is an interface, and as such would rightly not exist in the js file that's being referenced. In which case, it's probably something I can fix

**Reply by VividLemon** [reply]
Attempt with https://github.com/bootstrap-vue/bootstrap-vue-next/releases/tag/v0.7.1

**Reply by amery** [reply]
better, now `BodyProp` is missing :-) for `BToast.vue:21` @VividLemon 

```
Uncaught (in promise) SyntaxError: The requested module '/_nuxt/@fs/home/xxx/node_modules/.pnpm/bootstrap-vue-next@0.7.1_rplo2tn3ftbcpuocymkccezv2u/node_modules/bootstrap-vue-next/src/components/BToast/plugin.ts?v=0521ffcc' does not provide an export named 'BodyProp' (at BToast.vue:21:9)
```

**Reply by amery** [reply]
I would try to get them all fixed but it doesn't happen when I link a local checkout :( suggestions for a noob @VividLemon ?

**Reply by VividLemon** [reply]
If you're planning on trying to fix btoast, I'd recommend against that. It needs to go through a major refactor that needs some prerequisite work done first. Work that would take a deeper knowledge of the lib, bootstrap, and our plans for the future

**Comment by amery** [comment]
I'm not using BToast, but these issues prevent me from using your releases in Nuxt at all.
I thought that if I could "link" the local copy and still use it in the same js-way as if imported from the release and could submit a single commit adding all the missing `import type` entries

**Reply by VividLemon** [reply]
Toast, as well as a few others don't fail because of the types, they fail because of other reasons. Though they are going to start being worked on soon. Once I fix up a few last carousel issues. Try some suggestions from https://github.com/bootstrap-vue/bootstrap-vue-next/discussions/912#discussion-4812429
